CONTRIBUTING.rst

```diff
@@ -86,7 +86,7 @@ without using a local copy.  This can be convenient for small fixes.
 
         $ tox -e docs
 
-    The built documentation should be available in the ``doc/en/_build/``.
+    The built documentation should be available in ``doc/en/_build/html``.
 
     Where 'en' refers to the documentation language.
```

Comment:
> Where 'en' refers to the documentation language.

I was about to remove this sentence, and then also thought / got reminded of removing the "en" subdir.
It was useful for when there where other languages ("jp" IIRC), but I do not think it is planned to have other languages again?
This might require to setup redirections on RTD though likely, so is nothing too trivial.

Reply:
Yeah this already occurred to me, but I also not thought it worth the effort (as I don't know how to do the redirect and would need to research it), so I never pursued it.

Reply:
https://readthedocs.org/dashboard/pytest/redirects/